package.json

```diff
@@ -111,6 +111,19 @@
 					"default": [
 						"--local-kamelet-dir=."
 					]
+				},
+				"camel.debugAdapter.KubernetesRunParameters": {
+					"type": "array",
+					"uniqueItems": true,
+					"items": {
+						"type": "string"
+					},
+					"additionalProperties": false,
+					"markdownDescription": "User defined parameters to be applied at every deploy (See [Camel JBang Kubernetes](https://camel.apache.org/manual/camel-jbang-kubernetes.html)). In case of spaces, the values needs to be enclosed with quotes. Default value is `[\"--cluster-type=openshift\", \"--dev\"]`\n\n**Note**: Excluding `--camel-version` which is already being set in `#camel.debugAdapter.CamelVersion#`.\n\nFor more possible values see: `camel kubernetes run --help` or `jbang camel@apache/camel kubernetes run --help`",
```

Comment:
at the moment we cannot use by default `--dev` which is not working for Camel 4.8.1+ (see https://issues.apache.org/jira/browse/CAMEL-21388)